api/swagger.yaml

```diff
@@ -7274,18 +7276,17 @@ paths:
                 information about the path
         400:
           description: "Bad parameter"
-          schema:
-            allOf:
-              - $ref: "#/definitions/ErrorResponse"
-              - type: "object"
-                properties:
-                  message:
-                    description: |
-                      The error message. Either "must specify path parameter"
-                      (path cannot be empty) or "not a directory" (path was
-                      asserted to be a directory but exists as a file).
-                    type: "string"
-                    x-nullable: false
+          content:
+            application/json:
+              schema:
+                $ref: "#/definitions/ErrorResponse"
+              examples:
```

Comment:
I think there was something special with this (but could be due to the (older) version of Swagger / OpenAPI we're using);  with this change, the response doesn't show in the example responses;

before this change

<img width="779" alt="Screenshot 2022-03-07 at 15 13 20" src="https://user-images.githubusercontent.com/1804568/157050815-4583e055-589a-47b3-9143-32015bf8356f.png">


after this change:

<img width="762" alt="Screenshot 2022-03-07 at 15 13 27" src="https://user-images.githubusercontent.com/1804568/157050830-95e19ba7-a6db-4a47-8f02-fcd41d62efb2.png">

Reply:
This is what I wanted to check...
how do you generate this website so I can experiment with this?

Reply:
You can run `make swagger-docs` which will start a container with it at `http://localhost:9000`

Once we have these things fixed, it'd be good to also include (a) separate commit(s) to "backport" the fixes to the `docs/api/vXX.XX.yaml` files, which are used for the documentation of each version of the API.


At least v1.41 (current) would be good to update, but depending on how easy it applies to other versions, I usually do a quick compare, and update those as well.

Reply:
Updating redoc just allow to use multiple example as supported by [OpenAPI](https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.1.0.md#example-object-examples)

Reply:
If we do, we need to do the version used on docs.docker.com as well, and check if the layout still looks ok with it. ISTR they stopped providing a "bundle" version at some point (which is why I didn't update it there yet), but perhaps it can be downloaded from a different location; the script for the docs are in this directory; (js minified and a .map for debugging); https://github.com/docker/docker.github.io/tree/master/js

Reply:
this one is outdated, as I would have guessed :'(